src/modules/Dropdown/Dropdown.js

```diff
@@ -902,9 +906,7 @@ export default class Dropdown extends Component {
 
   scrollSelectedItemIntoView = () => {
     debug('scrollSelectedItemIntoView()')
-    // Do not access document when server side rendering
-    if (!isBrowser) return
-    const menu = document.querySelector('.ui.dropdown.active.visible .menu.visible')
```

Comment:
This fixes another bug.  If multiple Dropdowns were open, this would select the first one on the page.  This meant the Menu for a Dropdown not belonging to this one was being operated on.

I've fixed that by selecting _this_ Dropdown's Menu.